File: app/src/main/java/com/github/krot/game/Game.java
package com.github.krot.game;

import com.github.krot.sqlite.helper.DatabaseHelper;
import com.github.krot.utils.SystemProperties;

public class Game {

    private static Game instance;

    private Game() {
    }

    public static Game getInstance() {
        if (instance == null) {
            instance = new Game();
        }
        return instance;
    }

    public Round nextRound() {
        int nextRoundId = SystemProperties.NEXT_ROUND;
        return DatabaseHelper.getHelper().findRoundById(nextRoundId);
    }
}

File: app/src/main/java/com/github/krot/game/Game.kt
package com.github.krot.game

import com.github.krot.utils.RoundsProducer
import com.github.krot.utils.SystemProperties

object Game {
    var curValue = 0f
    var targetValue = 0f
    private val roundIdToRound = RoundsProducer.getRounds().map { it.id to it }.toMap()

    fun nextRound() = roundIdToRound[SystemProperties.NEXT_ROUND]
}
